packages/framework/agent-scheduler/src/test/tsconfig.json

```diff
@@ -1,14 +1,15 @@
 {
-	"extends": ["../../../../../common/build/build-common/tsconfig.test.json"],
+	"extends": "../../../../../common/build/build-common/tsconfig.test.node16.json",
+	"compilerOptions": {
+		"rootDir": "./",
+		"outDir": "../../lib/test",
+		"types": [],
+		"noEmit": true,
```

Comment:
curious why is `noEmit` is option added? : )

Reply:
The 'noEmit: true' tells TypeScript not to both producing files, since nothing is executed for type tests.